test/examples/Makefile.am

```diff
@@ -8,16 +8,15 @@
 
 examplesdir = $(pkgdatadir)/examples
 dist_examples_DATA = \
+	ucp_hello_world.c \
 	uct_hello_world.c \
 	active_message.c 
 
 installcheck-local:
 	@echo "INSTALLCHECK: Compiling examples with installed library"
-
-	$(CC) -o uct_hello_world $(examplesdir)/uct_hello_world.c -luct -lucs -I$(includedir) -L$(libdir)
-
+	$(CC) -o uct_hello_world $(examplesdir)/uct_hello_world.c -luct -I$(includedir) -L$(libdir)
```

Comment:
why not to use autotools way to specify *.c files? using specific gcc flags and way to compile examples is not very portable (remember - we want to dominate the world of HPC)